### PR TITLE
feat: ✨Implemented 5 Methods. 

### PR DESCRIPTION
**Pull request by kaumudpa**
Implemented getBlock, getTransaction, getTransactionConfirmations, getBlockTransactionCount, sendTransaction

Please review and Merge

### CODE REVIEW

**Comment by kaumudpa** [comment]
Closes #21

File: example/pubspec.yaml [hunk 1]
```diff
@@ -39,6 +39,7 @@ dependencies:
 
   wagmi_flutter_web:
     path: ../
+  # web3dart: ^2.7.3
```
Comment (redDwarf03) [inline example/pubspec.yaml:42]: Why this declaration of web3dart (just for info webthree is better :))

Comment (kaumudpa) [reply example/pubspec.yaml:42]: Sorry my bad! We were using it for testing purposes.

**Comment by redDwarf03** [comment]
Great PR. thanks

Just 2 things
- Could you please fix lints 
- If I'm not mistaken, i don't find the sendTransaction method in the example app

**Comment by kaumudpa** [comment]
I dont see any problems with lints - Can you guide me what is it you want me to do?

**Comment by redDwarf03** [comment]
Capture d’écran 2024-08-28 à 17 04 05

**Comment by kaumudpa** [comment]
Okay - Committing in few mins

**Comment by kaumudpa** [comment]
@redDwarf03 please check now and let Me know if its ok!

**Pull request merged**